examples/timers/tests/src/timers.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { resolve } from 'node:path';
+import { PocketIc, Actor } from '@dfinity/pic';
+
+import { _SERVICE, idlFactory } from '../../declarations/timers.did';
+
+const WASM_PATH = resolve(
+  __dirname,
+  '..',
+  '..',
+  '..',
+  '..',
+  '.dfx',
+  'local',
+  'canisters',
+  'timers',
+  'timers.wasm.gz',
+);
+
+describe('Timers', () => {
+  let pic: PocketIc;
+  let actor: Actor<_SERVICE>;
+
+  beforeEach(async () => {
+    pic = await PocketIc.create(process.env.PIC_URL);
+    const fixture = await pic.setupCanister<_SERVICE>({ idlFactory, wasm: WASM_PATH });
+    actor = fixture.actor;
+  });
+
+  afterEach(async () => {
+    await pic.tearDown();
+  });
+
+  it('Check trapping function', async () => {
+    try {
+      await actor.trapping();
+      throw new Error('Expected trapping() to throw');
+    } catch (err) {
+      const msg = (err as Error)?.message ?? String(err);
+      process.stdout.write(`${msg}\n`);
+      expect(msg).toContain('This is a trap message');
+    }
```

Comment:
This can be done more simply like this. I'd also suggest using snapshot testing on the console output here.

```suggestion
    await expect(actor.trapping()).rejects.toMatch('This is a trap message');
```